src/operations/expansions/expandGeom2.js

```diff
@@ -6,24 +6,31 @@ const offsetFromPoints = require('./offsetFromPoints')
  * Expand the given geometry (geom2) using the given options (if any).
  * @param {Object} options - options for expand
  * @param {Number} [options.delta=1] - delta (+/-) of expansion
- * @param {Integer} [options.segments=0] - number of segments when creating rounded corners, or zero for chamfer
+ * @param {String} [options.corners='round'] - type corner to create during of expansion; edge, chamfer, round
+ * @param {Integer} [options.segments=16] - number of segments when creating round corners
  * @param {geom2} geometry - the geometry to expand
  * @returns {geom2} expanded geometry
  */
 const expandGeom2 = (options, geometry) => {
   const defaults = {
     delta: 1,
-    segments: 0
+    corners: 'round',
```

Comment:
hmm I was gonna say 'all ok', but did we not say that 'edge' should be the default ? (as that has no approximations etc)

Reply:
Yeah. I was going to make ‘edge’ the default, but V1 implementation is ‘round’.

I’m fine with edge but thought a short discussion would be best.

Thoughts?

Reply:
Done. It's 'edge' now.